client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.component.ts

```diff
@@ -30,7 +30,8 @@ export class IncidentInfoPanel implements AfterViewInit {
   public constructor(private publishedIncidentService: PublishedIncidentService, private snackbarService: MatSnackBar, private appConfigService: AppConfigService,
                      private cdr: ChangeDetectorRef,
                      private router: ActivatedRoute,
-                     private http: HttpClient) {}
+                     private http: HttpClient,
+                     protected route: Router) {}
```

Comment:
private if it isn't referenced outside the class/html

Reply:
It is referenced in the child class. I wanted to import Router to the child class without creating its own implementation for each of the parent's constructor params. Is this the correct way to acheive that?

Reply:
Yep, that's fine.